Source/Extensions/Blazorise.ImageCropper/AspectRatio.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Blazorise.ImageCropper;
+
+/// <summary>
+/// Cropper aspect ratio
+/// </summary>
+public record struct AspectRatio( double? Value )
+{
+    /// <summary>
+    /// Create a new aspect ratio based on width and height
+    /// </summary>
+    /// <param name="width">ratio width</param>
+    /// <param name="height">ratio height</param>
+    public AspectRatio( int width, int height )
+        : this( (double)width / height )
+    {
+    }
+
+    /// <summary>
+    /// 16:9 aspect ratio
+    /// </summary>
+    public static AspectRatio Ratio16_9 = new( 16, 9 );
```

Comment:
Do you think better naming would be with an `x`? eg. `Ratio16x9 `?

Reply:
yes good suggestion

Reply:
Maybe even remove Ratio, and just have it as `Is16x9`.